.changeset/silly-dragons-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': minor
+---
+
+Adds return public-incoming-payment returnt type to incoming payment get request which should be returned for unauthenticated requests to this endpoint
```

Comment:
nit: typo on `returnt` -> `return`, if we don't change it here we'll have to change it on the bot PR.

Reply:
fixed